changelogs/fragments/8783-preserve-parameters-provided-from-task-in-zypper-module.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - keycloak_client - fix TypeError when sanitizing the ``saml.signing.private.key`` attribute in the module's diff or state output. The ``sanitize_cr`` function expected a dict where in some cases a list might occur (https://github.com/ansible-collections/community.general/pull/8403).
```

Comment:
This message is about another module and doesn't make sense for this one.

Reply:
Right, sorry, I had edited the changes but not saved them...